tests/CMakeLists.txt

```diff
@@ -42,5 +42,14 @@ kagen_add_test(test_generic_file_generator
     FILES file/generic_file_generator_test.cpp
     CORES 1 2 3 4 8 16)
 
+# Geometric Generator
+kagen_add_test(geometric_2d
+        FILES geometric/geometric_2d.cpp
```

Comment:
Rename file to `geometric_2d_test.cpp` etc. and target to `test_geometric_2d` etc.